matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/pushers/PushersService.kt

```diff
@@ -64,6 +62,25 @@ interface PushersService {
                       append: Boolean,
                       withEventIdOnly: Boolean): UUID
 
+    /**
+     * Add a new Email pusher.
+     * Ref: https://matrix.org/docs/spec/client_server/latest#post-matrix-client-r0-pushers-set
+     *
+     * @param email             The email address to send notifications to.
+     * @param lang              The preferred language for receiving notifications (e.g. "en" or "en-US").
+     * @param emailBranding     The branding placeholder to include in the email communications.
+     * @param appDisplayName    A human readable string that will allow the user to identify what application owns this pusher.
+     * @param deviceDisplayName A human readable string that will allow the user to identify what device owns this pusher.
+     * @return                  A work request uuid. Can be used to listen to the status
```

Comment:
the next PR will remove the reliance on the `WorkManager` by switching to a  `suspend fun`, allowing us to handle and simplify the registration errors